webapp/src/Controller/API/ProblemController.php

```diff
@@ -258,6 +258,137 @@ public function addProblemAction(Request $request): array
         ];
     }
 
+    /**
+     * Delete a problem from this contest.
```

Comment:
I wonder if we should call this unlink instead of delete?

Reply:
Sure, and call the other one link 😄

Reply:
Done.